### PR TITLE
Add legend to main bar.

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
see #158

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,9 +34,33 @@
...
+                  <input type="radio" name="display" id="other" value="other"><span>alle</span>
+		</label>
+		<!--
+		<label>
+		  <div class="awesome-marker-icon-darkpurple awesome-marker">
+		    <i class="icon icon-basket icon-white"></i>
+		  </div>
+		  <input type="radio" name="display" id="unclassified" value="unclassified"><span>sonstige</span>
+		</label>
+		-->
```
Comment (ubergesundheit) [inline index.html]: Remove this commented out part

**Comment by Ryuno-Ki** [comment]
@ubergesundheit Updated.

**Comment by ubergesundheit** [comment]
Thanks! If no other objections from @wo-ist-markt/web-devs is raised, this should be ready to be merged.

**Comment by k-nut** [comment]
Nice work! 
Two things though
1) The markers get cut off at the bottom a bit. Maybe we can make them slightly smaller by applying a `transform: scale(0.8)` to them?
2) I think it would actually be nice to have the fourth marker for the unclassified markets. Was there a reason why you did not add it?

**Comment by Ryuno-Ki** [comment]
@k-nut Agreed. I scaled down the icon a bit. Since it is interactive, I applied `cursor: pointer` to the whole `<label>`.

Regarding second: https://github.com/wo-ist-markt/wo-ist-markt.github.io/pull/312#discussion_r331184265 - I can easily revert this commit if needed.

**Comment by k-nut** [comment]
Nice I'll merge this as it is and then maybe you can just open a new PR that also adds a non-interactive section for the last marker type?

**Comment by Ryuno-Ki** [comment]
Note, that I haven't forgotten about your request (= have an item in my ToDo list), but dealing with life at the moment.

Do you have an idea, why this PR is flagged as pending in Hacktoberfest UI?

**Comment by Ryuno-Ki** [comment]
No, all of them. But the others aren't merged yet. Okay, I'm gonna be patient then.

**Pull request merged**